commons-core/src/main/scala/com/avsystem/commons/serialization/TransparentWrapperCompanion.scala

```diff
@@ -43,3 +43,6 @@ abstract class TransparentWrapperCompanion[R, T] extends TransparentWrapping[R,
 abstract class StringWrapperCompanion[T] extends TransparentWrapperCompanion[String, T]
 abstract class IntWrapperCompanion[T] extends TransparentWrapperCompanion[Int, T]
 abstract class LongWrapperCompanion[T] extends TransparentWrapperCompanion[Long, T]
+abstract class FloatWrapperCompanion[T] extends TransparentWrapperCompanion[Float, T]
```

Comment:
Are you srsly using `Float`?

Reply:
Not really, needed `Double` but added this for consistency.